### DISCUSSION TITLE
Please support TOML for hugo-friendly front matter

### DISCUSSION THREAD

**Discussion by chicks-net** [Zettlr/Zettlr#5288] [Ideas]
I see that YAML is support for front matter when it is delimited by `---`.  Could TOML delimited by `+++` also be supported?

**Comment by the-solipsist** [comment]
Given that Zettlr uses Pandoc in the background, which in turn uses YAML frontmatter (and `---`), I don't think this would be desirable.  Given that [Hugo also supports YAML frontmatter](https://gohugo.io/content-management/front-matter/), why not just use that?

**Reply by chicks-net** [reply]
I'm a much bigger fan of TOML than YAML so it was an easy choice for my personal site since hugo fully supports it.  I'm happy to continue encouraging the adoption of markup languages that are human friendly (such as Markdown and TOML).  

I understand that you're leaving these questions to pandoc so TOML isn't a feature that you're likely to implement within this project.

**Reply by chicks-net** [reply]
https://github.com/jgm/pandoc/issues/10295 for anyone interested in following along in the pandoc project.

**Reply by nathanlesage** [reply]
I tend to agree with John, Albert, and Caleb in the Pandoc threat you linked, and with @the-solipsist here: since basically any Markdown compatible parser nowadays supports YAML, including Hugo, I don't see it anywhere on the roadmap for now. Once adoption becomes more serious, which I doubt in the coming years, we may reopen the discussion.
